mipdb/commands.py

```diff
@@ -34,83 +32,24 @@
 from mipdb.usecases import UntagDataset
 from mipdb.usecases import ValidateDataset
 
+CONFIG = "/home/config.toml"
```

Comment:
`CONFIG_PATH` ?

Also, if you want to change this for dev environment, you could read this path from an env variable, with a default value if it doesn't exist.